extensions/eclipse-che-theia-plugin-remote/src/browser/remote-plugin-command.ts

```diff
@@ -0,0 +1,32 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { CommandRegistry, CommandContribution, Command } from '@theia/core/lib/common';
+import { RemotePluginStarterService } from '../common/remote-plugin-protocol';
+
+const COMMAND: Command = {
+    id: '_remote_plugin_start_'
+};
+
+@injectable()
+export class RemotePluginCommandContribution implements CommandContribution {
+
+    @inject(RemotePluginStarterService)
+    private starterService: RemotePluginStarterService;
+
+    registerCommands(commands: CommandRegistry): void {
+        commands.registerCommand(COMMAND, {
+            execute: () => {
+                this.starterService.loadRemotePlugins();
```

Comment:
It's better to have
```
starterService.start()
```
It it's `starterService` or `pluginStarter` then better to name it `start`, if it's `pluginLoader` then `load`.

Reply:
Also the user can run this command using command palette. Could you prevent that by adding `isEnabled()` and `isVisible()` methods to command handler? Something like below
```
commands.registerCommand(COMMAND, {
  let initialized: boolean = false;
  execute: () => {
    initialized = true;
  },
  isEnabled() {
    return !initialized;
  },
  isVisible() {
    return !initialized;
  }
```